.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:16-buster-slim
+# install base development tools
+RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync chromium parallel python3
+
+# make /bin/sh run /bin/bash
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN npm install -g jest
```

Comment:
remove this

Reply:
without "jest", the vscode-jest extension is not loading.

Reply:
Jest is installed in node_modules when you run `yarn`. I don't have jest globally on my machine so we shouldn't install It globally in the container.

Reply:
removed it